common/presenters/move-type-for-filter.js

```diff
@@ -0,0 +1,9 @@
+const i18n = require('../../config/i18n')
+const { cloneDeep } = require('lodash')
+
+function moveTypeForFilter(item) {
```

Comment:
Could we keep the consistency in naming convention for these - `moveTypesToFilterComponent`? It was something that was suggested by a previous developer to try and be verbose and clear about what the presenters are doing and I think it works well.

Reply:
ok